dts/nios2/nios2-qemu.dtsi

```diff
@@ -31,10 +31,10 @@
 		compatible = "simple-bus";
 		ranges;
 
-		uart0: uart@201000 {
```

Comment:
cant we have both enabled? This way we can merge and start testing with new qemu and do the switch back and forth if needed. I would leave jtag until qemu is available.

Reply:
Ok will keep the jtag option as well